app/models/equipment_listing.rb

```diff
@@ -1,5 +1,5 @@
 class EquipmentListing < ApplicationRecord
-  belongs_to :owner, class_name: "User"
```

Comment:
In here, owner refers to User but we called it owner so we can differentiate between renters and owners. 
Payla and I did this on Tuesday? but since it is new to us we may need to ask a TA to check it.

![image](https://user-images.githubusercontent.com/61854797/180091821-0fd482e2-67b1-4fb6-b545-a7d921f5f614.png)